src/app.ts

```diff
@@ -97,7 +97,7 @@ const apiRoutes: Array<RouteHandler<ApiLocation>> = [
   },
   {
     location: trustMemberApiLocation,
-    handler: membersRoutes.trust(membersCollection, operationsCollection)
+    handler: membersRoutes.trust(db, membersCollection, operationsCollection)
```

Comment:
Hmm, maybe we should just make getting the `membersCollection` and `operationsCollection` a function that takes in `db`, since these are sorta redundant

Reply:
Makes sense, created #51 to track.